docs/js-core-2/week-2/homework.md

```diff
@@ -4,6 +4,32 @@ title: Coursework
 sidebar_label: Coursework
 ---
 
+## 0) Flipped Classroom Practice (3h)
+
+1.1 Given the following array of objects
+
+```js
+let people = [
+  { name: "Bob", age: 30 },
+  { name: "John", age: 20 },
+  { name: "Elise", age: 35 },
+];
+```
+
+Log to the console an array containing the people older than 25
+Log to the console an array containing the names of the people
+
+```
+[ { name: 'Bob', age: 30 }, { name: 'Elise', age: 35 } ]
+[ 'Bob', 'John', 'Elise' ]
+```
+
+2.1 TDD practice
+
+Fork and clone https://github.com/CodeYourFuture/tdd-practice. Begin solving the exercise by answering the questions about the first few steps.
```

Comment:
I like the tdd practice repo: however, I think the `README` is too detailed and prescriptive at the moment. I think at most it should state the first 2 test descriptions and then prompt the trainees to think about how they should implement the test cases and then the actual logic for the function itself. This will be challenging: but they need to wrestle with the process a bit to really get into it. Your `README` at the moment is more like a solution guide.